ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/ip-nat.cfg

```diff
@@ -61,16 +61,16 @@ interface Management1
    vrf MGMT
    ip address 10.73.255.122/24
 !
-ip nat pool prefix_16 prefix-length 16
-   range 10.0.0.1 10.0.255.254
-   range 10.1.0.0 10.1.255.255 1024 65535
-   utilization threshold 1 action log
-ip nat pool prefix_32 prefix-length 32
-   range 10.2.0.1 10.2.0.1
 ip nat pool prefix_24 prefix-length 24
-   range 10.3.0.1 10.3.0.254
-   range 10.3.1.0 10.3.1.255 1024 65535
    utilization threshold 100 action log
+ip nat pool prefix_32 prefix-length 32
```

Comment:
i am trying but unable to get the config when i need only ips for prefix-length. can you add an example in molecule?

```
ip nat pool ABX prefix-length 20 
   range 1.1.1.1 1.1.1.2
```

Reply:
I missed this part, thanks for checking this :)

Reply:
Added!